scripts/install

```diff
@@ -158,11 +159,10 @@ cp ../conf/plugins.local.php            $final_path/conf/plugins.local.php.bak
 
 # Create file if it does not exist
 if [ ! -f "$final_path/conf/local.protected.php" ]; then
```

Comment:
Hmm, I can't seem to figure out how this case can happen, considering line 117...?

Reply:
Good catch. Copy pasta from upgrade script I guess. Will remove that part